src/components/Header/Header.jsx

```diff
@@ -0,0 +1,63 @@
+import linkbraryLogo from "images/logo.svg";
+import { useEffect, useState } from "react";
+import FolderInfo from "components/FolderInfo/FolderInfo";
+import Profile from "components/Profile/Profile";
+import { getFolder, getProfile } from "api";
+import "./Header.css";
+
+function Header() {
+  const [email, setEmail] = useState("");
+  const [profileImageSource, setProfileImageSource] = useState(null);
+  const [hasProfile, setHasProfile] = useState(false);
+  const [folderName, setFolderName] = useState("");
+  const [ownerName, setOwnerName] = useState("");
+  const [ownerProfileImageSource, setOwnerProfileImageSource] = useState(null);
+
+  const handleLoadProfile = async () => {
+    const result = await getProfile();
+    if (!result) {
+      setHasProfile(false);
+      return;
+    }
+    const { email, profileImageSource } = result;
+    setEmail(email);
+    setProfileImageSource(profileImageSource);
+    setHasProfile(true);
+  };
+
+  const handleLoadFolder = async () => {
+    const result = await getFolder();
+    const { folder } = result;
+    const { name, owner } = folder;
```

Comment:
오호 혹시 이렇게 destruction했을때 에러 안나나요?
아마 folder가 undefined일 경우 destruction하지 못한다고 에러가 발생될수도있어요.
이 에러를 미연에 방지하기 위해

```suggestion
    const name = folder?.name;
    const owner = folder?.owner;
```

이렇게 바꿔주는것을 추천드립니다!